packages/core/src/components/image-overlay/image-overlay.scss

```diff
@@ -0,0 +1,74 @@
+@import "~@dso-toolkit/sources/src/styling/variables";
+
+@import "../../icon/di";
+
+@import "~@dso-toolkit/sources/src/styling/global/button";
+@import "~@dso-toolkit/sources/src/styling/mixins/box-sizing";
+@import "~@dso-toolkit/sources/src/styling/mixins/sr-only";
+
+@import "~@dso-toolkit/sources/src/components/button/button";
+
+$dso-image-overlay-z-index: 20;
+
+:host {
+  display: inline-block;
+  position: relative;
+}
+
+:host(:hover) .open,
+.open:focus {
+  opacity: 1;
```

Comment:
@benheldoorn Bij mij bleef de test `.should(.not.be.visible')` falen omdat Cypress niet zag dat ie met `right: 99999px'` niet buiten beeld viel. Ik begrijp niet waarom jij daar geen last van had. Ik heb het in ieder geval opgelost met `opacity`, voorzie jij problemen?

Reply:
Nee